src/snovault/auditor.py

```diff
@@ -244,8 +244,9 @@ def item_view_audit(context, request):
     path = request.resource_path(context)
     properties = request.embed(path, '@@object')
     inherit = context.audit_inherit
-    if context.embedded and '*' in context.audit_inherit:
-        inherit = context.embedded
+    embedded_paths = list(context.embedded) + [p.path for p in context.embedded_with_frame]
```

Comment:
There are a couple places where we have to iterate over these paths. Think I could make this cleaner by just having a property that implements this logic once instead of leaking internals every time we have to do this.